console/web3ext/web3ext.go

```diff
@@ -716,6 +716,17 @@ web3._extend({
 			params: 2,
 			inputFormatter: [web3._extend.formatters.inputAddressFormatter, web3._extend.formatters.inputDefaultBlockNumberFormatter],
 		}),
+		new web3._extend.Method({
```

Comment:
`klay_getBlock` supports a number and a hash as well. How about adding `klay_getHeader`?

```
var blockCall = function (args) {
    return (utils.isString(args[0]) && args[0].indexOf('0x') === 0) ? "klay_getBlockByHash" : "klay_getBlockByNumber";
};

var getBlock = new Method({
    name: 'getBlock',
    call: blockCall,
    params: 2,
    inputFormatter: [formatters.inputBlockNumberFormatter, function (val) { return !!val; }],
    outputFormatter: formatters.outputBlockFormatter
});
```

Reply:
I added getHeader!
Thank you!